assets/sass/components/_header-bar.scss

```diff
@@ -1,7 +1,7 @@
 #header-contents {
   display: flex;
   align-items: center;
-  justify-content: space-between;
+  justify-content: flex-start;
```

Comment:
This is so `main-title` is next to `hmpps-title` rather than in the centre which is what happens with `space-between`